### PR TITLE
PLAT-10873 PLAT-10828: Added more examples in the Python BDK bot

### PR DESCRIPTION
**Pull request by symphony-elias**
### Ticket
PLAT-10873
PLAT-10828

### Description
Added examples of: slash command, CommandActivity, FormReplyActivity and UserJoinedRoomActivity
Added example of Jinja2 usage

### Checklist
- [x] Referenced a ticket in the PR title and in the corresponding section
- [x] Filled properly the description and dependencies, if any
- [N/A] Public functions properly commented

### CODE REVIEW

File: generators/bdk/python/templates/bot-app/activities.py.ejs [hunk 1]
```diff
@@ -0,0 +1,39 @@
+from symphony.bdk.core.activity.command import CommandActivity, CommandContext
+from symphony.bdk.core.activity.user_joined_room import UserJoinedRoomActivity, UserJoinedRoomContext
+from symphony.bdk.core.service.message.message_service import MessageService
+from symphony.bdk.core.service.user.user_service import UserService
+
+
+class EchoCommandActivity(CommandActivity):
```
Comment (symphony-youri) [inline generators/bdk/python/templates/bot-app/activities.py.ejs:7]: would it be simpler as a SlashCommandActivity ?

File: generators/bdk/python/templates/bot-app/activities.py.ejs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    def __init__(self, messages: MessageService):
+        self._messages = messages
+        super().__init__()
+
+    def matches(self, context: CommandContext) -> bool:
+        return context.text_content.startswith("@" + context.bot_display_name + " " + self.command_name)
+
+    async def on_activity(self, context: CommandContext):
+        text_to_echo = context.text_content[context.text_content.index(self.command_name) + len(self.command_name):]
+        await self._messages.send_message(context.stream_id, f"<messageML>{text_to_echo}</messageML>")
```
Comment (symphony-youri) [inline generators/bdk/python/templates/bot-app/activities.py.ejs:21]: we could even drop the <messageML> part now right?

File: generators/bdk/python/templates/requirements.txt.ejs [hunk 3]
```diff
@@ -1 +1,2 @@
 sym-api-client-python>=<%= bdkVersion %>
+Jinja2
```
Comment (symphony-youri) [inline generators/bdk/python/templates/requirements.txt.ejs:2]: don't we have to set a version here?

File: generators/bdk/python/templates/bot-app/activities.py.ejs (hunk 1 above)
Comment (symphony-elias) [reply generators/bdk/python/templates/bot-app/activities.py.ejs:7]: I wanted to exemplify the use of a CommandActivity (not a SlashCommand)

File: generators/bdk/python/templates/requirements.txt.ejs (hunk 3 above)
Comment (symphony-elias) [reply generators/bdk/python/templates/requirements.txt.ejs:2]: No version means pip installs the latest version.

**Review by symphony-youri — APPROVED** [review]

**Pull request merged**